lib/components/narrative/default/itinerary.css

```diff
@@ -1,6 +1,6 @@
 /* ITINERARY OPTION */
 
-.otp .option.default-itin button:focus {
+.otp .option.default-itin :not(button) {
```

Comment:
Wait I don't think this rule is valid. I was thinking something more along the lines of 

`.otp .option.default-itin button:not(.other-itin):focus`

Reply:
This works, thank you!!